openmmml/models/anipotential.py

```diff
@@ -63,27 +63,35 @@ def addForces(self,
                   atoms: Optional[Iterable[int]],
                   forceGroup: int,
                   filename: str = 'animodel.pt',
+                  use_OptimizedTorchANI : Optional[bool] = True, 
```

Comment:
Let's rename it to `useOptimizedTorchANI`.  It's mixing two different naming conventions in a single name.